src/client/coll/results.rs

```diff
@@ -0,0 +1,104 @@
+use bson;
+use bson::Bson;
+
+use std::collections::BTreeMap;
+
+/// Results for a bulk write operation.
+#[derive(Clone)]
+pub struct BulkWriteResult {
```

Comment:
Since `Result` means something in Rust, this is a bit odd. I'm not sure if that's enough to figure out a better name, and I'm not sure what name to suggest...